src/control/content/components/ProgressBar.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
Is this the same progress bar as on Digital Member Id?
The plugin is using bootstrap and this seems not to follow that convention

Reply:
It's not the same progress bar as on Digital member ID, this one I made, because I wasn't able to link actual progress to the bootstrap one without introducing additional external packages. Hope this works. This one actually fills the progress in real time and with actual progression of the import